movement/analysis/kinematics.py

```diff
@@ -165,3 +169,175 @@ def compute_time_derivative(data: xr.DataArray, order: int) -> xr.DataArray:
     for _ in range(order):
         result = result.differentiate("time")
     return result
+
+
+def compute_forward_vector(
+    data: xr.DataArray,
+    left_keypoint: str,
+    right_keypoint: str,
+    camera_view: Literal["top_down", "bottom_up"] = "top_down",
+):
+    """Compute a 2D forward vector given two left-right symmetrical keypoints.
+
+    The forward vector is computed as a vector perpendicular to the
+    line connecting two symmetrical keypoints on either side of the body
+    (i.e., symmetrical relative to the mid-sagittal plane), and pointing
+    forwards (in the rostral direction). A top-down or bottom-up view of the
+    animal is assumed.
+
+    To determine the forward direction of the animal, we need to specify
+    (1) the right-to-left direction of the animal and (2) its upward direction.
+    We determine the right-to-left direction via the input left and right
+    keypoints. The upwards direction, in turn, can be determined by passing the
+    ``camera_view`` argument with either ``"top_down"`` or ``"bottom_up"``. If
```

Comment:
Minor quibble, but I think using both monospace and quotation for the top-down/bottom-up is overkill. I get what you are trying to express here (show that the argument values is a string), but I think it's sufficient to just say "top-down"/"bottom-up" (without monospace). The argument names (e.g `camera view`) can keep being monospace, as in other docstrings. This obviously concerns all such occurrences within the new docstrings.

I know I'm partly to blame, for not having compiled an "Editorial style guide" yet...

Anyhow, feel free to ignore my take on this, it's entirely subjective. This docstring is complete as is.

Reply:
Now that I've looked at the rendered output in the docs, I quite like the look, so 🤷🏼 . I leave the decision to you, just make sure that it's consistently done throughout the 2 new docstrings.